internal/nodejs-action/action.yml

```diff
@@ -51,10 +68,21 @@ runs:
       shell: bash
       run: ./../__TOOL_ACTION_DIR__/build.sh
 
+    - id: sha
+      uses: slsa-framework/slsa-github-generator/.github/actions/compute-sha256@main
+      with:
+        path: ${{ steps.build.outputs.file-path }}
+
+    # rng generates a random number to avoid name collision in artifacts
+    # when multiple workflows run concurrently.
+    - name: Generate random 16-byte value (32-char hex encoded)
+      id: rng
+      uses: slsa-framework/slsa-github-generator/.github/actions/rng@main
```

Comment:
so we'll need to expose this function to TRWs, it seems.
It's an interesting and very important use case, I added it to the BYOB doc list https://github.com/slsa-framework/slsa-github-generator/issues/1574#issuecomment-1405878671

Reply:
I was thinking a bit about this. I think we should think more about this. I'm not sure we need to provide a lot of "utilities" to TRW authors or we should provide better ways to pass out info about the artifacts & attestations that are uploaded by the internal action.

Reply:
I think #1863 covers this pretty well and we won't really need this if we implement that idea.